assets/dev/scss/frontend/_container.scss

```
@@ -101,8 +101,8 @@
 
 	&-boxed.e-grid {
 		justify-items: initial;
-		grid-template-columns: initial;
-		grid-template-rows: initial;
+		grid-template-columns: 1fr;
+		grid-template-rows: 1fr;
```

```suggestion
		grid-template-rows: repeat(2, 1fr);
```
Just making sure? This is the default according to the spec.